### PR TITLE
Profile pictures

### PR DESCRIPTION
**Pull request by ocao29**
## Link to Issue
Profile Pictures

## Description of changes
I updated WorkerLandingPage.tsx to call the backend to load client's profile photos. I also added a test for ClientProfilePage.tsx to ensure that photos are being saved to the backend when uploaded.

## Screenshot(s) or GIF(s) of changes
Screen Shot 2022-06-23 at 12 12 18 PM

### CODE REVIEW

File: src/components/LandingPages/WorkerLanding.tsx [hunk 1]
```diff
@@ -38,6 +39,10 @@ interface State {
   showClients: boolean;
   currentPage: number;
   postsPerPage: number;
+<<<<<<< HEAD
```
Comment (crchong1) [inline src/components/LandingPages/WorkerLanding.tsx:42]: merge conflict here

File: src/tests/components/AccountSettings/ClientProfilePage.test.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    fetchMock.resetMocks();
+  });
+
+  describe('Upload Profile Photo', () => {
+    test('should call backend to upload photo', async () => {
+      // Arrange
+      fetchMock.mockResponseOnce(JSON.stringify({ status: 'SUCCESS' }));
+
+      const alertShowFn = jest.fn();
+      const file = new File(['profile photo'], 'photo.png', { type: 'image/png' });
```
Comment (crchong1) [inline src/tests/components/AccountSettings/ClientProfilePage.test.tsx:27]: feel free to use an actual image from the src/static/img folder here just so you are simulating a real image

File: src/components/LandingPages/WorkerLanding.tsx [hunk 3]
```diff
@@ -181,10 +205,56 @@ class WorkerLanding extends Component<Props, State> {
...
+          username,
+        }),
+      })
+          .then((response) => response.blob());
+    });
+
+    return Promise.all(promises).then((results) => {
+      photos = results.map((blob) => {
+        const { size } = blob;
+        if (size > 72) {
```
[the remark this answers is not in the document]
Comment (ocao29) [reply src/components/LandingPages/WorkerLanding.tsx:230]: That was working code from earlier that I just kept. I can adjust it so there's no magic number, but do you want it to be refactored entirely?

**Comment by ocao29** [comment]
I checked with the server code from April 24 and the frontend does work properly. If this pull request looks good, I think it's okay to merge, we just need to go back and fix the backend.
Screen Shot 2022-06-30 at 2 05 56 PM

**Comment by lizyn** [comment]
The last version of backend code that works for me is 78d2a409dbd5ca2f2a8a15ab2602461add4e4307, and things break starting at 9a0c2d5efa4c06f3d785337333cefcec047b80c0 

image

**Review by lizyn — APPROVED** [review]

**Pull request merged**